crates/egui/src/widget_text.rs

```diff
@@ -67,6 +67,27 @@ impl From<String> for RichText {
     }
 }
 
+impl From<&Box<str>> for RichText {
+    #[inline]
+    fn from(text: &Box<str>) -> Self {
+        Self::new(text.clone())
+    }
+}
+
+impl From<&mut Box<str>> for RichText {
+    #[inline]
+    fn from(text: &mut Box<str>) -> Self {
+        Self::new(text.clone())
+    }
+}
+
```

Comment:
Is there a reason `&mut Box<str>` is implemented for RichText but not for WidgetText?

Reply:
I followed what already exists for `String`.

I believe Rust will [coerce](https://doc.rust-lang.org/reference/type-coercions.html) &mut anyway.